components/Templates/includes/auto-template/Pods_Templates_Auto_Template_Front_End.php

```diff
@@ -279,12 +319,27 @@ public function front( $content ) {
 		// now use other methods in class to build array to search in/ use
 		$possible_pods = $this->auto_pods();
 
+		// build Pods object for current item
+		global $post;
+		$pod_name = $current_post_type;
+		$pod_item = $post->ID;
+		if ( in_the_loop() ) {
+			$pod_name = $post->post_type;
+		} else {
+			// Outside the loop in a taxonomy, we want the term
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 9bc72b684a4cea8ff85eb01486d70cf2</sub>